scripts/packetHandlers.ts

```diff
@@ -1,160 +1,269 @@
-			Call.infoMessage(con, "also keep in mind that T H E R E  I S   A   P A C K E T   S P A M   L I M I T");
-			Call.infoMessage(con, "[green]Also keep in mind that you have to multiply by 8 to spawn it at a clear coordinate (For example instead of 5,5 you'd have to do 5*8,5*8 to spawn a thing at 5,5, does this sound confusing...)");
-			Call.infoMessage(con, `"worst error handling i have ever seen, why kick the player???"\n  -ASimpleBeginner`);
-			Call.infoMessage(con, `"The code style when submitted was beyond drunk... but it worked... barely " -BalaM314`);
-			Call.infoMessage(con, `these packet handlers and everything related to them were made by [green]frog`);
-			Call.infoMessage(con, "All commands mentioned should be performed on the client side console");
-			if(sender.hasPerm("bulkLabelPacket")){
-				Call.infoMessage(con, `bulkLabel - Call.serverPacketReliable("bulkLabel",/*it's basically like label but seperated by | you get the idea*/) - this is admin only`);
-			}
-			Call.infoMessage(con, `label - Call.serverPacketReliable("label",[text,duration,x,y].join(","))\nthe text cannot be larger than 41 characters, duration cannot be larger than 10`);
-			Call.infoMessage(con, `lineEffect - Call.serverPacketReliable("lineEffect",[startX,startY,endX,endY,color].join(","))\n The color is a hex code and a string`);
-			Call.infoMessage(con, `bulkLineEffect - Call.serverPacketReliable("bulkLineEffect",[startX,startY,endX,endY,color].join(",")+"|"+[startX,startY,endX,endY,color].join(",")+"|"))) - lineEffect but seperated by | so packet spam won't be a problem, can only contain 10 effects`);
+		handler({sender, output}) {
+			const responseLines:string[] = [];
+			const canBulk:boolean = sender.hasPerm('bulkLabelPacket');
+			
+			responseLines.push('Line effect: "lineEffect", "x0,y0,x1,y1,hexColor" (for example "20.7,19.3,50.4,28.9,#FF0000")\n');
+			if (canBulk) responseLines.push('Bulk line effect: "bulkLineEffect", equivalent to multiple lineEffect packets, with every line separated by a \'|\' symbol.\n');
+			responseLines.push('Label effect: "label", "content,duration,x,y" (for example ""Hi!",10,20,28")\n');
+			if (canBulk) responseLines.push('Bulk label effect: "bulkLabel", equivalent to multiple label packets, with every label separated by a \'|\' symbol.\n');
+			
+			responseLines.push('Use "Call.serverPacketReliable" to send these.');
+			responseLines.push('You need to multiply world coordinates by Vars.tilesize (8) for things to work properly. This is a relic from the v3 days where every tile was 8 pixels.');
+			responseLines.push('Keep in mind there\'s a packet spam limit. Use at your own risk.');
+
+			responseLines.push(''); //empty line
 
+			//credit
+			responseLines.push('These packet handlers and everything related to them were made by [green]frog[white].');
+			responseLines.push('Most of the code was rewritten in 2024 by [#6e00fb]D[#9e15de]a[#cd29c2]r[#fd3ea5]t[white].');
```

Comment:
move this line after the complaints (it sounds like I was complaining about your code, which is actually good)

Reply:
done